Sources/Afterpay/Afterpay.swift

```diff
@@ -418,7 +423,9 @@ internal var brand: Brand {
 }
 
 public var enabled: Bool {
-  return language != nil && getConfiguration()?.locale != nil
+  let enabledByV2Config = language != nil && getConfiguration()?.locale != nil
+  let enabledByV3Config = language != nil  && getV3Configuration()?.region == .US
+  return enabledByV2Config || enabledByV3Config
```

Comment:
I like this idea. One thought is that some of the views (ie `PriceBreakdownView`) rely on this enabled flag - and depend on the primary configuration (`getConfiguration`) being set.

This may require a separate flag - perhaps `checkoutEnabled` which does the same thing and replaces `enabled` in all checkouts OR `v3Enabled` which just handles the V3 Checkout.